modules/kargoBidAdapter.js

```diff
@@ -172,28 +172,6 @@ export const spec = {
     return spec._getCrbFromCookie();
   },
 
-  _getKruxUserId() {
-    return spec._getLocalStorageSafely('kxkar_user');
-  },
-
-  _getKruxSegments() {
-    return spec._getLocalStorageSafely('kxkar_segs');
-  },
-
-  _getKrux() {
-    const segmentsStr = spec._getKruxSegments();
-    let segments = [];
-
-    if (segmentsStr) {
-      segments = segmentsStr.split(',');
-    }
-
-    return {
-      userID: spec._getKruxUserId(),
-      segments: segments
-    };
-  },
-
   _getLocalStorageSafely(key) {
     try {
       return storage.getDataFromLocalStorage(key);
```

Comment:
fyi, bidders will be disabled from accessing local storage by default in prebid 7